web/src/components/header.tsx

```diff
@@ -53,23 +80,95 @@ const H2 = styled.h2`
   padding: 0px 4px;
   text-decoration: none;
   font-size: 1.54912em;
+  @media (max-width: 600px) {
+    font-size: 1em;
+  }
 `;
 
-const Container = styled.div`
-  display: flex;
-  font-size: 1em;
-`;
+interface INavProps {
+  name: LayoutSet;
+  isDDMobileOpen: boolean;
+  handleDDMobile: (e: any) => void;
+  dirs: Array<MenuItem>;
+}
+
+const Nav: React.FC<INavProps> = props => {
+  const { name, handleDDMobile, isDDMobileOpen, dirs } = props;
+
+  const dropdownMenuID: string = 'DropdownCreatebutton';
+
+  const handleDDDesktop = (event: any) => {
+    event.stopPropagation();
+    document?.getElementById(dropdownMenuID)?.classList.toggle(DROPDOWN_CLASSNAME);
```

Comment:
The question mark before a var is for telling Typescript in local dev env this var can be undefined. Otherwise, it wouldn't compile /  convert to regular JS I think. I could be wrong.

Reply:
ohh interesting! I didn't know that

Reply:
I wonder if the `.tsconfig` has a way of declaring global objects